crates/sui-core/src/node_sync/node_state.rs

```diff
@@ -657,23 +668,28 @@ where
         }
     }
 
-    async fn wait_for_parents(
+    /// Transaction initialized by Node Sync passes in a Semaphore for
+    /// concurrency controlling while Transaction initialized by the write path
```

Comment:
will everyone know what "while Transaction initialized by the write path..." means? What is the "write path"?